core/src/avm2/regexp.rs

```diff
@@ -2,9 +2,13 @@
 
 use std::borrow::Cow;
 
+use crate::avm2::activation::Activation;
+use crate::avm2::Error;
+use crate::avm2::{ArrayObject, ArrayStorage, Object};
 use crate::string::{AvmString, Units, WStrToUtf8};
 use bitflags::bitflags;
 use gc_arena::Collect;
+use ruffle_wstr::WString;
```

Comment:
Currently Ruffle always imports `WString` using:
```rs
use crate::string::WString;
```
Let's do this as well for consistency.